lib/bolt/bolt_option_parser.rb

```diff
@@ -54,6 +54,15 @@ def get_help_text(subcommand, action = nil)
           { flags: OPTIONS[:global],
             banner: INVENTORY_HELP }
         end
+      when 'cache'
```

Comment:
This command needs to be added to the default banner text (`bolt --help`): https://github.com/puppetlabs/bolt/blob/main/lib/bolt/bolt_option_parser.rb#L187